week-1/fix/median.test.js

```diff
@@ -19,8 +19,8 @@ describe("calculateMedian", () => {
 
   test("doesn't modify the input", () => {
```

Comment:
Your code will pass this test, just make sure that line 22 isn't commented so it will run properly